### DISCUSSION TITLE
Redis CROSSSLOT keys request don't hash to the same slot.

### DISCUSSION THREAD

**Discussion by dfoley84** [apache/superset#28606] [Q&A / Help]
Hi All, 

Unable to find info on this error, I'm currently testing a ElastiCache Serverless deployment within AWS 

but when trying to restart Superset i'm getting the following Error 

`> packages/redis/client.py", line 1275, in parse_response response = connection.read_response() File "/usr/local/lib/python3.9/site-packages/redis/connection.py", line 882, in read_response raise response redis.exceptions.ResponseError: CROSSSLOT Keys in request don't hash to the same slot`

Current Superset_config Settings for Redis

```

DATA_CACHE_CONFIG = CACHE_CONFIG


class CeleryConfig(object):

    BROKER_URL = f"{REDIS_TYPE_DB}://{REDIS_HOST}:{REDIS_PORT}/{REDIS_CELERY_DB}?ssl_cert_reqs=CERT_NONE"
    CELERY_IMPORTS = (
        "superset.sql_lab", 
       "superset.tasks.scheduler",
    )
    CELERY_RESULT_BACKEND = f"{REDIS_TYPE_DB}://{REDIS_HOST}:{REDIS_PORT}/{REDIS_RESULTS_DB}?ssl_cert_reqs=CERT_NONE"
    CELERYD_LOG_LEVEL = "INFO"
    CELERYD_PREFETCH_MULTIPLIER = 10
    CELERY_ACKS_LATE = True
    CELERY_TASK_PROTOCOL = 1

    CELERY_ANNOTATIONS = {
        'sql_lab.get_sql_results': {
            'rate_limit': '100/s',
        },
    }


```

**Comment by oshriza** [comment]
I have the same issue. 
AWS ElastiCache serverless (redis cache) is cluster mode enabled, so redis cache db and celery should be in the same db and this cause the error.
I tried to use hash tag for them but without success.
Hope someone will come with a solution for this, or another solution of deploying with redis cache serverless
